app/templates/_base_page.html

```diff
@@ -1,48 +1,73 @@
-{% extends "toolkit/layouts/_base_page.html" %}
+{% extends "template.njk" %}
 
 {# Import GOV.UK Frontend components for globale usage#}
 {% from "components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {% from "components/button/macro.njk" import govukButton %}
 {% from "components/phase-banner/macro.njk" import govukPhaseBanner %}
 {% from "components/tabs/macro.njk" import govukTabs %}
 
-{# This can be removed once we start using govuk-frontend template #}
-{# The reason calling this block here is to change the header title so that it does not
-   include the phase tag #}
-{% block inside_header %}
-  <div class="header-title">
-    <a href="/">Digital Marketplace</a>
-  </div>
-{% endblock %}
+{% set assetPath = '/admin/static' %}
 
-{# This can be removed once we start using govuk-frontend template #}
-{# The reason for calling this is to ensure we do not output the user research signup banner #}
-{% block after_header %}
+{% block head %}
+  {% include "layouts/_custom_dimensions.html" %}
+  {% include "layouts/_site_verification.html" %}
+  <link type="text/css" rel="stylesheet" href="{{ asset_fingerprinter.get_url('stylesheets/application.css') }}" />
 {% endblock %}
 
-{# override content block to add a beforeContent block like the one in govuk-frontend template #}
-{% block content %}
-  {% block top_header %}{% endblock %}
-  <div id="wrapper">
-    {% block beforeContent %}
-      {{ govukPhaseBanner({
-        "tag": {
-          "text": "beta"
-        },
-        "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="'  + url_for('external.help') + '">send your feedback</a>'
-      }) }}
-      {% block breadcrumbs %}{% endblock %}
-    {% endblock %}
-
-    <main id="content" role="main">
-      {% include "toolkit/flash_messages.html" %}
-      {% block main_content %}
-      {% endblock %}
-    </main>
-  </div>
+{% block header %}
+
+  {% set headerNavigation = [
+    {
+      'text': 'Guidance',
+      'href': 'https://www.gov.uk/guidance/buying-and-selling-on-the-digital-marketplace'
```

Comment:
Do we have a `url_for` equivalent?

Reply:
We don't have these for external (including GOV.UK) links. 

The list of Flask views that are available across all the frontend apps is kept in utils: https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/external.py

Reply:
Might be an interesting tech debt ticket to add views to external that redirect to commonly used GOV.UK links; keep them all in one place and all.

Reply:
FYI we have a mix of framework-specific URLs (buried in the `frameworks` repo with the rest of the content, in theory easier to manage but certainly less visible when debugging) and framework-agnostic ones (which usually just get plonked in the templates, so easier to find but easier to forget about).